.pylintrc

```diff
@@ -117,7 +117,7 @@ max-module-lines=1500
 # separator` is used to allow tabulation in dicts, etc.: {1  : 1,\n222: 2}.
 # `trailing-comma` allows a space between comma and closing bracket: (a, ).
 # `empty-line` allows space-only lines.
-no-space-check=trailing-comma,dict-separator
+# no-space-check=trailing-comma,dict-separator
```

Comment:
Same as qnebe-613 - why commented out? Can we delete them, or hope to eventually get them working again?

Reply:
Deleted them, these settings were removed from pylint